### DISCUSSION TITLE
Which request policy should I use with `@urql/exchange-persisted`?

### DISCUSSION THREAD

**Discussion by predaytor** [urql-graphql/urql#3281] [Q&A]
I have Varnish setup with 10 minutes for `Cache-Control` (using https://github.com/wp-graphql/wp-graphql-smart-cache`). Should I use the `'network-only'` request policy instead of the default `'cache-first'`? Does `cacheExchange` have any in-memory TTL?
```ts
import { cacheExchange, createClient, fetchExchange, ssrExchange } from '@urql/core';
import { persistedExchange } from '@urql/exchange-persisted';

export const client = createClient({
	url: import.meta.env.WORDPRESS_API_URL,
	exchanges: [
		cacheExchange,
		ssrExchange({
			isClient: !import.meta.env.SSR,
		}),
		persistedExchange({
			preferGetForPersistedQueries: true,
		}),
		fetchExchange,
	],
	requestPolicy: 'network-only',
});
```

**Comment by predaytor** [comment]
up

**Reply by soleo** [reply]
@predaytor I was running into issues with caching combined with automatic persisted queries https://github.com/urql-graphql/urql/issues/3417

My recommendation for you is to remove both `cacheExchange` and `ssrExchange` to see if it would work correctly with your setup. 

```
import { cacheExchange, createClient, fetchExchange, ssrExchange } from '@urql/core';
import { persistedExchange } from '@urql/exchange-persisted';

export const client = createClient({
	url: import.meta.env.WORDPRESS_API_URL,
	exchanges: [
		persistedExchange({
			preferGetForPersistedQueries: true,
		}),
		fetchExchange,
	],
});
```

**Reply by predaytor** [reply]
thx for reply. I actually have no issues with automatic persisted queries so far. I kept the current configuration with a `network-only` request policy to rely only on the HTTP layer, bypassing Urql's internal caching.

**Reply by JoviDeCroock** [reply]
We have no TTL no we only work with automatic invalidation through mutations
